flink-autoscaler/src/main/java/org/apache/flink/autoscaler/ScalingExecutor.java

```diff
@@ -225,10 +227,14 @@ Map<JobVertexID, ScalingSummary> computeScalingSummary(
                             } else {
                                 var currentParallelism =
                                         (int) metrics.get(ScalingMetric.PARALLELISM).getCurrent();
+
+                                final boolean hasKeyBy =
+                                        jobTopology.get(v).getInputs().containsValue("HASH");
```

Comment:
It's a bit strange to check string values like `HASH` can we make this an enum so that it's more explicit in the autoscaler code and easier to handle?